src/include/txn_inline.h

```diff
@@ -483,7 +490,7 @@ __wt_txn_modify(WT_SESSION_IMPL *session, WT_UPDATE *upd)
     WT_ASSERT(session, !WT_IS_HS((S2BT(session))->dhandle));
 
     upd->txnid = session->txn->id;
-    __wt_txn_op_set_timestamp(session, op);
+    __wt_txn_op_set_timestamp(session, op, false);
```

Comment:
General question:
Can we pass in the timestamp to use instead of a boolean?

Reply:
It is confusing if we do that, because this function is also called for prepared case.